onnxruntime/core/framework/kernel_registry_manager.h

```diff
@@ -79,11 +79,17 @@ class KernelRegistryManager {
                       const IExecutionProvider& execution_provider,
                       SessionState& session_state,
                       const KernelCreateInfo& kernel_create_info, std::unique_ptr<OpKernel>& out) const;
-
+#ifdef _WIN32
+#pragma warning(push)
+// disable some warnings from protobuf to pass Windows build
```

Comment:
how is this warning from protobuf?
if it is from this code, can we fix it?

Reply:
I will revert this change and see how it goes. Sorry I forgot what warning I saw.